app/utils.ts

```diff
@@ -67,6 +67,13 @@ export function buildUserBaseDomain(username: PrismaUser['username']) {
   return `${cleanUsername(username)}.${process.env.ROOT_DOMAIN}`;
 }
 
+export function buildDomain(username: PrismaUser['username'], name?: string) {
```

Comment:
You can rebase to use what @dadolhay already landed in https://github.com/DevelopingSpace/starchart/commit/6719cee3c2d0d1d5d29022c6dc04d8fa1f7322e1#diff-9ba22a4ee607f76b338e5793ffbb2eae933b1b432aba0fbcd3a3fd14c9efbb1bR66